### ISSUE TITLE
glib_probes.h not found while building on Ubuntu 14.04

### ISSUE THREAD

**Issue by wangheda** [deepmind/lab#8]
I installed all the dependencies as told, cleaned and re-built a few times, and I still got this message:

```
wangheda@wangheda-HeyStack:~/Programs/DeepMindLab/lab$ bazel build :deepmind_lab.so --define headless=glx
INFO: Found 1 target...
ERROR: /home/wangheda/.cache/bazel/_bazel_wangheda/9a2a0f97587d2d4f1381984736a5dc83/external/glib_archive/BUILD:35:1: C++ compilation of rule '@glib_archive//:glib' failed: linux-sandbox failed: error executing command /home/wangheda/.cache/bazel/_bazel_wangheda/9a2a0f97587d2d4f1381984736a5dc83/execroot/lab/_bin/linux-sandbox ... (remaining 52 argument(s) skipped).
In file included from external/glib_archive/glib/gslice.c:54:0:
external/glib_archive/glib/glib_trace.h:33:25: fatal error: glib_probes.h: No such file or directory
 #include "glib_probes.h"
                         ^
compilation terminated.
Target //:deepmind_lab.so failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 0.875s, Critical Path: 0.68s
```

Also I found that in this directory:
`/home/wangheda/.cache/bazel/_bazel_wangheda/9a2a0f97587d2d4f1381984736a5dc83/external/glib_archive/glib`
there is no glib_probes.h, only glib_probes.d which seems like a header file.

Any idea how to fix this?

**Comment by heiner** [comment]
Interesting. I'm not able to reproduce this one on Ubuntu 14.04:

```
heiner@ubuntu-trusty-1:~/lab$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.5 LTS"
heiner@ubuntu-trusty-1:~/lab$ bazel build :deepmind_lab.so --define headless=glx
INFO: Found 1 target...
Target //:deepmind_lab.so up-to-date:
  bazel-bin/deepmind_lab.so
INFO: Elapsed time: 20.433s, Critical Path: 16.86s
```

To resolve this, we'll probably need more information about your system: GCC version, Bazel version and how you installed it, etc.

**Comment by wangheda** [comment]
Here's my system information:
```
wangheda@wangheda-HeyStack:~$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.5 LTS"
```

Bazel version:
```
wangheda@wangheda-HeyStack:~$ bazel version
Build label: 0.4.2
Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Wed Dec 7 18:47:11 2016 (1481136431)
Build timestamp: 1481136431
Build timestamp as int: 1481136431
```

GCC version:
```
wangheda@wangheda-HeyStack:~$ gcc --version
gcc (Ubuntu 4.8.4-2ubuntu1~14.04.3) 4.8.4
Copyright (C) 2013 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Python version:
```
wangheda@wangheda-HeyStack:~/Programs/DeepMindLab/lab$ python --version
Python 2.7.6
```

I installed Bazel using custom APT repository. And I followed the build instructions to apt-get install the dependencies. Then I cloned the code repository, and ran bazel build.

If I start as new, I would get an output as the attached [error.txt](https://github.com/deepmind/lab/files/646552/error.txt).

**Comment by wangheda** [comment]
I did the following things and problems was resolved:

$ sudo apt-get remove systemtap-sdt-dev
$ bazel clean
$ bazel build :deepmind_lab.so --define headless=glx

I have no idea why it works. I did so because of [this thread](https://groups.google.com/forum/#!topic/openbricks-devel/4bsPeXGjDwA).

**Comment by tkoeppe** [comment]
Excellent, thank you for the update!

**Comment by decster** [comment]
try to fix this in #34
